docs/user/tutorials/installing.md

```diff
@@ -0,0 +1,31 @@
+# Installing Hypper
+
+## From source (Linux, macOS)
+
+Building Hypper from source is a bit more work, but the best way to test the
+latest (pre-release) Hypper version.
+
+You must have a working Go environment (1.14+).
+
+```terminal
+$ git clone https://github.com/rancher-sandbox/hypper/hypper.git
+$ cd hypper
+$ make
+```
+
+If required, it will fetch the dependencies and cache them, and validate
+configuration.
+It will then compile `hypper`, and place it in `./bin/hypper`.
+
+
+## From development builds
+
+In addition to releases you can download and install development snapshots of
+Hypper from [Github Actions].
+
+## From Github releases
```

Comment:
As soon as this is decided, this should be the first item in this document.

Reply:
I would not be surprised if we have a get script to handle the install. We want to make it easy. In any case, this section will get rewritten at least once.